scripts-dev/add_admin.py

```diff
@@ -0,0 +1,39 @@
+import requests
+import json
+import sqlite3
+import logging
+
+
+def reg_admin(cursor):
+    headers = {'content-type': 'application/json'}
+    url = 'http://localhost:8008/_matrix/client/api/v1/register'
+    post_data = {'user': 'admin',
+                 'password': 'admin123',
+                 'type': 'm.login.password'}
```

Comment:
Hi, 
Made a new commit.
1- Fix all error from flake8
2- add docstring to all the new methods
3 - Used _simple_select_list_paginate as private method which is more generic to use for other list search as well and used get_user_list_paginate for calling from rest api